v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/spanner/ProcessInformationSchema.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.spanner;
+
+import com.google.cloud.spanner.BatchClient;
+import com.google.cloud.spanner.BatchReadOnlyTransaction;
+import com.google.cloud.spanner.DatabaseAdminClient;
+import com.google.cloud.spanner.Dialect;
+import com.google.cloud.spanner.TimestampBound;
+import com.google.cloud.teleport.v2.spanner.ddl.Ddl;
+import com.google.cloud.teleport.v2.spanner.ddl.InformationSchemaScanner;
+import org.apache.beam.sdk.io.gcp.spanner.SpannerAccessor;
+import org.apache.beam.sdk.io.gcp.spanner.SpannerConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Beam transform which reads information schema using {@link InformationSchemaScanner} and outputs
+ * the updated information schema.
+ */
+public class ProcessInformationSchema {
+  private static final Logger LOG = LoggerFactory.getLogger(ProcessInformationSchema.class);
+
+  public static Ddl getInformationSchemaAsDdl(SpannerConfig spannerConfig) {
```

Comment:
This method just constructs the correct inputs(which are all 3rd party calls), and calls scanner scan. Unit testing this would require mocking pretty much every line of code, there is no logic we are actually testing here. Feels like high effort, very low reward to do this.